config/packager.php

```diff
@@ -7,6 +7,7 @@
      * Default: http://github.com/Jeroen-G/packager-skeleton/archive/master.zip
      */
     'skeleton' => 'http://github.com/Jeroen-G/packager-skeleton/archive/master.zip',
+    'cache_skeleton' => false,
```

Comment:
What would be a proper place to document this? I'm thinking a line in just the config file might be enough?

Reply:
Now I wonder if it's even worth keeping. The only benefit is that you save about a second of download time (if even that). On the other hands, it adds a layer of complexity, and people might end up using an outdated skeleton without knowing it.